frontend/src/store/slices/index.ts

```diff
@@ -4,3 +4,4 @@ export { reducer as authReducer, AuthActionCreator } from './auth/auth.slice';
 export { reducer as clinicsReducer, ClinicsActionCreator } from './clinics/clinics.slice';
 export { reducer as mapReducer, MapActionCreator } from './map/map.slice';
 export { reducer as profileReducer, ProfileActionCreator } from './profile/profile.slice';
+export { reducer as appointmentReducer, AppointmentActionCreator } from './appointment/appointment.slice';
```

Comment:
Rename it to the `doctor-details` and move all connected logic (make appointment and current doctor functionality)